### ISSUE TITLE
依照範例定義training data 卻還是發生root ERROR: No Images in train dataset錯誤

### ISSUE THREAD

**Issue by julian7862** [PaddlePaddle/PaddleOCR#3835]
[2021/08/27 15:46:18] root INFO: Architecture :
[2021/08/27 15:46:18] root INFO:     Backbone :
[2021/08/27 15:46:18] root INFO:         model_name : large
[2021/08/27 15:46:18] root INFO:         name : MobileNetV3
[2021/08/27 15:46:18] root INFO:         scale : 0.5
[2021/08/27 15:46:18] root INFO:     Head :
[2021/08/27 15:46:18] root INFO:         fc_decay : 0.0004
[2021/08/27 15:46:18] root INFO:         name : CTCHead
[2021/08/27 15:46:18] root INFO:     Neck :
[2021/08/27 15:46:18] root INFO:         encoder_type : rnn
[2021/08/27 15:46:18] root INFO:         hidden_size : 96
[2021/08/27 15:46:18] root INFO:         name : SequenceEncoder
[2021/08/27 15:46:18] root INFO:     Transform :
[2021/08/27 15:46:18] root INFO:         loc_lr : 0.1
[2021/08/27 15:46:18] root INFO:         model_name : small
[2021/08/27 15:46:18] root INFO:         name : TPS
[2021/08/27 15:46:18] root INFO:         num_fiducial : 20
[2021/08/27 15:46:18] root INFO:     algorithm : STARNet
[2021/08/27 15:46:18] root INFO:     model_type : rec
[2021/08/27 15:46:18] root INFO: Eval :
[2021/08/27 15:46:18] root INFO:     dataset :
[2021/08/27 15:46:18] root INFO:         data_dir : ./train_data/
[2021/08/27 15:46:18] root INFO:         label_file_list : ['./train_data/train_sample.txt']
[2021/08/27 15:46:18] root INFO:         name : SimpleDataSet
[2021/08/27 15:46:18] root INFO:         transforms :
[2021/08/27 15:46:18] root INFO:             DecodeImage :
[2021/08/27 15:46:18] root INFO:                 channel_first : False
[2021/08/27 15:46:18] root INFO:                 img_mode : BGR
[2021/08/27 15:46:18] root INFO:             CTCLabelEncode : None
[2021/08/27 15:46:18] root INFO:             RecResizeImg :
[2021/08/27 15:46:18] root INFO:                 image_shape : [3, 32, 320]
[2021/08/27 15:46:18] root INFO:             KeepKeys :
[2021/08/27 15:46:18] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/08/27 15:46:18] root INFO:     loader :
[2021/08/27 15:46:18] root INFO:         batch_size_per_card : 256
[2021/08/27 15:46:18] root INFO:         drop_last : False
[2021/08/27 15:46:18] root INFO:         num_workers : 4
[2021/08/27 15:46:18] root INFO:         shuffle : True
[2021/08/27 15:46:18] root INFO: Global :
[2021/08/27 15:46:18] root INFO:     cal_metric_during_train : True
[2021/08/27 15:46:18] root INFO:     character_dict_path : ppocr/utils/dict/chinese_cht_dict.txt
[2021/08/27 15:46:18] root INFO:     character_type : ch
[2021/08/27 15:46:18] root INFO:     checkpoints : None
[2021/08/27 15:46:18] root INFO:     debug : False
[2021/08/27 15:46:18] root INFO:     distributed : False
[2021/08/27 15:46:18] root INFO:     epoch_num : 72
[2021/08/27 15:46:18] root INFO:     eval_batch_step : [0, 2000]
[2021/08/27 15:46:18] root INFO:     infer_img : doc/imgs_words_en/word_10.png
[2021/08/27 15:46:18] root INFO:     infer_mode : False
[2021/08/27 15:46:18] root INFO:     log_smooth_window : 20
[2021/08/27 15:46:18] root INFO:     max_text_length : 25
[2021/08/27 15:46:18] root INFO:     pretrained_model : None
[2021/08/27 15:46:18] root INFO:     print_batch_step : 10
[2021/08/27 15:46:18] root INFO:     save_epoch_step : 3
[2021/08/27 15:46:18] root INFO:     save_inference_dir : None
[2021/08/27 15:46:18] root INFO:     save_model_dir : ./output/rec/mv3_tps_bilstm_ctc/
[2021/08/27 15:46:18] root INFO:     save_res_path : ./output/rec/predicts_mv3_tps_bilstm_ctc.txt
[2021/08/27 15:46:18] root INFO:     use_gpu : True
[2021/08/27 15:46:18] root INFO:     use_space_char : False
[2021/08/27 15:46:18] root INFO:     use_visualdl : False
[2021/08/27 15:46:18] root INFO: Loss :
[2021/08/27 15:46:18] root INFO:     name : CTCLoss
[2021/08/27 15:46:18] root INFO: Metric :
[2021/08/27 15:46:18] root INFO:     main_indicator : acc
[2021/08/27 15:46:18] root INFO:     name : RecMetric
[2021/08/27 15:46:18] root INFO: Optimizer :
[2021/08/27 15:46:18] root INFO:     beta1 : 0.9
[2021/08/27 15:46:18] root INFO:     beta2 : 0.999
[2021/08/27 15:46:18] root INFO:     lr :
[2021/08/27 15:46:18] root INFO:         learning_rate : 0.0005
[2021/08/27 15:46:18] root INFO:     name : Adam
[2021/08/27 15:46:18] root INFO:     regularizer :
[2021/08/27 15:46:18] root INFO:         factor : 0
[2021/08/27 15:46:18] root INFO:         name : L2
[2021/08/27 15:46:18] root INFO: PostProcess :
[2021/08/27 15:46:18] root INFO:     name : CTCLabelDecode
[2021/08/27 15:46:18] root INFO: Train :
[2021/08/27 15:46:18] root INFO:     dataset :
[2021/08/27 15:46:18] root INFO:         data_dir : ./train_data/
[2021/08/27 15:46:18] root INFO:         label_file_list : ['./train_data/train_sample.txt']
[2021/08/27 15:46:18] root INFO:         name : SimpleDataSet
[2021/08/27 15:46:18] root INFO:         transforms :
[2021/08/27 15:46:18] root INFO:             DecodeImage :
[2021/08/27 15:46:18] root INFO:                 channel_first : False
[2021/08/27 15:46:18] root INFO:                 img_mode : BGR
[2021/08/27 15:46:18] root INFO:             CTCLabelEncode : None
[2021/08/27 15:46:18] root INFO:             RecResizeImg :
[2021/08/27 15:46:18] root INFO:                 image_shape : [3, 32, 320]
[2021/08/27 15:46:18] root INFO:             KeepKeys :
[2021/08/27 15:46:18] root INFO:                 keep_keys : ['image', 'label', 'length']
[2021/08/27 15:46:18] root INFO:     loader :
[2021/08/27 15:46:18] root INFO:         batch_size_per_card : 256
[2021/08/27 15:46:18] root INFO:         drop_last : True
[2021/08/27 15:46:18] root INFO:         num_workers : 8
[2021/08/27 15:46:18] root INFO:         shuffle : True
[2021/08/27 15:46:18] root INFO: train with paddle 2.1.2 and device CUDAPlace(3)
[2021/08/27 15:46:18] root INFO: Initialize indexs of datasets:['./train_data/train_sample.txt']
root ERROR: No Images in train dataset, please ensure
        1. The images num in the train label_file_list should be larger than or equal with batch size.
        2. The annotation file and path in the configuration file are provided normally.

由於先測試能否training只擺放一張照片跟一個label_file 如下
paddleOCR
-----train_data
        ----------train.png
        ----------train_sample.txt
yml參數為
    name: SimpleDataSet
    data_dir: ./train_data/
    label_file_list: ["./train_data/train_sample.txt"]

train_sample.txt裡面為
train.png \t train

找了好久都不知道錯誤在哪
已經依照示範的做過好多種可能
有沒有一個現在確定可以訓練的版本

**Comment by guoxiaoyue111111** [comment]
您好，请问您的问题解决了吗？

**Comment by vtmjapandev** [comment]
您好，请问您的问题解决了吗？
